package.json

```diff
@@ -6,21 +6,26 @@
 		"mlive": "bin/mlive"
 	},
 	"repository": {
-	      	"type": "git",
-	      	"url": "https://github.com/joyent/manta-mlive"
+		"type": "git",
+		"url": "https://github.com/joyent/manta-mlive"
 	},
-	"keywords": [ "manta", "liveness" ],
+	"keywords": [
+		"manta",
+		"liveness"
+	],
 	"author": "David Pacheco <dap@joyent.com>",
 	"license": "MPL-2.0",
-	"bugs": { "url": "https://github.com/joyent/manta-mlive/issues" },
+	"bugs": {
+		"url": "https://github.com/joyent/manta-mlive/issues"
+	},
 	"homepage": "https://github.com/joyent/manta-mlive",
 	"dependencies": {
 		"assert-plus": "^1.0.0",
 		"bunyan": "^1.5.1",
 		"extsprintf": "^1.3.0",
-		"manta": "^4.1.1",
+		"manta": "git://github.com/joyent/node-manta#buckets",
```

Comment:
this line seems less than ideal

Reply:
Yea not great I don't suppose. Are we doing this because the buckets work is still on a branch over in node-manta?

Reply:
Yep, that's the main reason. for now... i assume before buckets go live we should have the buckets branch merged into master with actual release numbers.